### DISCUSSION TITLE
Bug Report

### DISCUSSION THREAD

**Discussion by the-meeping** [FakeFishGames/Barotrauma#15342] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

the opaque and transparent line of sight options both seem to use the same exact effect, and im pretty sure thats not intended considering they are different options.

here are some comparison screenshots, the only difference between them i notice is the voltage in the lighting.

![t1](https://github.com/user-attachments/assets/ddd70c93-1402-4051-ac1c-ac1415b60b47)
![t2](https://github.com/user-attachments/assets/bf9bbceb-d03a-43a2-93a3-172233a41b0d)

### Reproduction steps

i was just looking at it in the sub editor, and i was resetting the sub test between changes (because i noticed that was enough to reload the line of sight effect without needing to fully reset the game)

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
you can vaguely see the outline of rooms in transparent mode

**Reply by Regalis11** [reply]
Yep, this is correct. The amount of ambient light in the level determines how dark the "line of sight" effect is (how clearly you can see through the walls), so in most levels it's still close to black.
